src/main/java/io/javalin/core/JavalinServlet.kt

```diff
@@ -54,61 +55,51 @@ class JavalinServlet(
         res.characterEncoding = defaultCharacterEncoding
         res.contentType = defaultContentType
 
-        try { // before-handlers, endpoint-handlers, static-files
-
-            for (beforeEntry in matcher.findEntries(HandlerType.BEFORE, requestUri)) {
-                beforeEntry.handler.handle(ContextUtil.update(ctx, beforeEntry, requestUri))
+        catchException(ctx) {
```

Comment:
mapExceptions ?

Reply:
Done lower, see `catchException` definition.

Reply:
Yes, I'm just suggesting to rename it to `mapException`

Reply:
Maybe move it to the exceptionMapper itself